Tests/Integration/MobileCoinClientPublicApiIntTests.swift

```diff
@@ -1,538 +1,12 @@
-
-                XCTAssertTrue(
-                    mobBalance.amountParts.int > 0 ||
-                    mobBalance.amountParts.frac > 0
-                )
-
-                guard let mobUSDBalance = balances.balances[.MOBUSD] else {
-                    XCTFail("Expected Balance")
-                    return
-                }
-
-                XCTAssertTrue(
-                    mobUSDBalance.amountParts.int > 0 ||
-                    mobUSDBalance.amountParts.frac > 0
-                )
-
-                let unknownTokenId = TokenId(UInt64(17000))
-                XCTAssertNil(balances.balances[unknownTokenId])
-
-                completion(balances)
-            }
-        }
-
-        func verifyBalanceChange(
-                _ client: MobileCoinClient,
-                _ balancesBefore: Balances,
-                _ expect: XCTestExpectation
-                ) {
-
-            var numChecksRemaining = 5
-            func checkBalanceChange() {
-                numChecksRemaining -= 1
-                print("Updating balance...")
-                client.updateBalances {
-                    guard let balances = $0.successOrFulfill(expectation: expect) else { return }
-                    print("Balances: \(balances)")
-
-                    do {
-                        let balancesMap = balances.balances
-                        let balancesBeforeMap = balancesBefore.balances
-                        let mobUSD = try XCTUnwrap(balancesMap[.MOBUSD]?.amount())
-                        let initialMobUSD = try XCTUnwrap(balancesBeforeMap[.MOBUSD]?.amount())
-
-                        guard mobUSD != initialMobUSD else {
-                            guard numChecksRemaining > 0 else {
-                                XCTFail("Failed to receive a changed balance. initial balance: " +
-                                    "\(initialMobUSD), current balance: " +
-                                    "\(mobUSD) microMOBUSD")
-                                expect.fulfill()
-                                return
-                            }
-
-                            Thread.sleep(forTimeInterval: 2)
-                            checkBalanceChange()
-                            return
-                        }
-                    } catch {}
-                    expect.fulfill()
-                }
-            }
-            checkBalanceChange()
-        }
-
-        try IntegrationTestFixtures.createMobileCoinClientWithBalance(
-                expectation: expect,
-                transportProtocol: transportProtocol) { client in
-            checkBlockVersionAndFee(client, expect) { fee in
-                checkBalances(client, expect) { balancesBefore in
-                    prepareAndSubmit(client, expect, fee) {
-                        verifyBalanceChange(client, balancesBefore, expect)
-                    }
-                }
-            }
-        }
-    }
-
-    func testCreateSignedContingentInput() throws {
```

Comment:
Ah, right... in the first pass I had left this off because the create is included in the submit SCI test (i.e.  an SCI needs to be created to submit).

I can leave this one in for now, like the RTH tests, and then move them both to async (was holding off on the RTH one originally b/c it wasn't passing).

I'll get the RTH one passing with the test data, migrate the RTH and create SCI tests to async, and make a note here when done.

Reply:
* RTH Test Data was updated manually at first
* Moved RTH test to Async
* Logic added to attempt a pre-population of RTH test data (MemoTypes), test will fail if all types not subsequently present
* Renamed MobileCoinClientPublicAsyncApiIntTests.swift -> MobileCoinClientPublicApiIntTests.swift (i.e. removed "Async") as the previous MobileCoinClientPublicApiIntTests.swift was empty